packages/fxa-payments-server/server/lib/amplitude.js

```diff
@@ -57,6 +57,12 @@ function getLocation(request) {
     return {};
   }
 
+  if(geodbConfig?.locationOverride?.location) {
+    return {
+      countryCode: geodbConfig.locationOverride.location.countryCode
+    };
```

Comment:
Can we return `locationOverride.location` here? We don't want to assume they only care about overriding the `countryCode`, even though that's currently the only thing we're using this `getLocation` helper for.

Note: This will error locally with the default `locationOverride`, since we don't have the MaxMind service running locally, and we'll still be calling `geolocate`, but I agree with you that I think that's what we want the default behavior to be.

Reply:
done